Makefile.am

```diff
@@ -154,6 +154,8 @@ cpplint_FILES += $(noinst_HEADERS)
 clean-local:
 	-rm -f $(BUILT_SOURCES) $(CLEANFILES)
 
+check: cpplint
+
```

Comment:
Is this correct, or should it be check-local?  I'm never sure with automake.  Possibly also redirect stdout of cpplint to avoid the spew of "processing x" messages that are mostly uninteresting?